src/Query/Wildcard.php

```diff
@@ -23,13 +23,14 @@ class Wildcard extends AbstractQuery
      */
     private $field;
 
-    public function __construct(string $field, string $value, float $boost = 1.0)
+    public function __construct(string $field, string $value, float $boost = 1.0, bool $caseInsensitive = false)
```

Comment:
Should probably not be passed as argument to constructor as it's optional.
As an example `rewrite` option is not passed as argument here.

Reply:
@ktherage I completely missed this, my bad
→ makes sense to remove it from the constructor, as for the "rewrite" parameter

Reply:
@thePanz, removed :wink: